### DISCUSSION TITLE
What is going on here?

### DISCUSSION THREAD

**Discussion by Genius398** [anuraghazra/github-readme-stats#1228] [Q&A] [answered]
Hi! I am Genius398, a coder. I code in HTML, CSS, JS and JSON. I would like to contribute here. If someone can just brief me about what happens here?

**Comment by mrhrifat** [comment, marked as answer]
Actually it's a Open Source Project to show different stats of any GitHub user profile based on his/her activities. To contribute here you should first fork this repository. Then make any changes like develop, fix something that is missing & which will make it better. Then create an pull request. If the owner of this project do like your commits, I think he will do merge.

**Reply by Genius398** [reply]
Thanks!
